website/src/components/SearchPage/SearchFullUI.tsx

```diff
@@ -450,6 +454,13 @@ export const InnerSearchFullUI = ({
                                 metadata={schema.metadata}
                                 richFastaHeaderFields={schema.richFastaHeaderFields}
                             />
+                            {linkOuts !== undefined && linkOuts.length > 0 && (
+                                <LinkOutMenu
```

Comment:
It would be good to improve the view on small screens. It now looks like this:

![image](https://github.com/user-attachments/assets/bd07c350-ba21-4d9b-b089-fa3d4d59a9e6)

(Before, it looked fine:

![image](https://github.com/user-attachments/assets/9aee12a2-a874-4a83-a6e0-3e88ebaf2ced))